src/WebJobs.Extensions.SendGrid/Config/SendGridConfiguration.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using Client;
+using Microsoft.Azure.WebJobs.Extensions.Bindings;
+using Microsoft.Azure.WebJobs.Extensions.Config;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using SendGrid.Helpers.Mail;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SendGrid
+{
+    /// <summary>
+    /// Defines the configuration options for the SendGrid binding.
+    /// </summary>
+    public class SendGridConfiguration : IExtensionConfigProvider
+    {
+        internal const string AzureWebJobsSendGridApiKeyName = "AzureWebJobsSendGridApiKey";
```

Comment:
same `[AppSetting(Default=...)]` comment as twilio

Reply:
Good suggestion. I think I'll get the initial migrations checked in before refactoring things.